CMakeLists.txt

```diff
@@ -1,5 +1,6 @@
 cmake_minimum_required(VERSION 3.20)
 
+option(OpenDDWStatic "OpenDDWStatic" OFF) #Build openddw to be compatible with a static build of opendds
```

Comment:
Instead of setting this, you could use [`OPENDDS_STATIC`](https://opendds.readthedocs.io/en/latest-release/building/cmake.html#var-OPENDDS_STATIC).

Reply:
Is there a way to get this flag from a complied opendds library or are you suggesting I just name the option consistently?

Reply:
The former, I think. `OPENDDS_STATIC` is a CMake variable, but if by define you mean a C predecessor macro, then you could use `ACE_AS_STATIC_LIBS`.